.github/workflows/locales-coverage.yml

```diff
@@ -0,0 +1,27 @@
+name: Build locales percentages
+
+on:
+  push:
+    paths:
+      - "src/locales/**.json"
+      - "!src/locales/percentages.json"
+
+jobs:
+  locales:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - name: Create report file
+        run: |
+          node scripts/build-locales-coverage.js
```

Comment:
Let's add a new script in the `package.json` and run it through there, so we can also test/run it easier locally.